part-1/5.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="utf-8" />
+<title>Land of Bump</title>
+<meta name="msapplication-TileColor" content="#ffffff" />
+<meta name="msapplication-TileImage" content="../ms-icon-144x144.png" />
+<meta name="theme-color" content="#ffffff" />
+<link rel="apple-touch-icon" sizes="57x57" href="../apple-icon-57x57.png" />
+<link rel="apple-touch-icon" sizes="60x60" href="../apple-icon-60x60.png" />
+<link rel="apple-touch-icon" sizes="72x72" href="../apple-icon-72x72.png" />
+<link rel="apple-touch-icon" sizes="76x76" href="../apple-icon-76x76.png" />
+<link rel="apple-touch-icon" sizes="114x114" href="../apple-icon-114x114.png" />
+<link rel="apple-touch-icon" sizes="120x120" href="../apple-icon-120x120.png" />
+<link rel="apple-touch-icon" sizes="144x144" href="../apple-icon-144x144.png" />
+<link rel="apple-touch-icon" sizes="152x152" href="../apple-icon-152x152.png" />
+<link rel="apple-touch-icon" sizes="180x180" href="../apple-icon-180x180.png" />
+<link rel="icon" type="image/png" sizes="192x192"  href="../android-icon-192x192.png" />
+<link rel="icon" type="image/png" sizes="32x32" href="../favicon-32x32.png" />
+<link rel="icon" type="image/png" sizes="96x96" href="../favicon-96x96.png" />
+<link rel="icon" type="image/png" sizes="16x16" href="../favicon-16x16.png" />
+<link rel="manifest" href="../manifest.json" />
+<link rel="stylesheet" href="../css/default.css" />
+<script>
+  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
+
+  ga('create', 'UA-6899787-52', 'auto');
+  ga('send', 'pageview');
+
+</script>
+</head>
+<body>
+<noscript><p class="warning">This page requires JavaScript to function properly. Please enable JavaScript in your browser.</p></noscript>
+<div id="page-container">
+  <div id="stage">
+    <video id="video1" poster="img/story/5.jpg" loop>
```

Comment:
I think this path should be "../img/story/5.jpg". The image isn't showing up when I view the page in a browser.

Reply:
Oh good catch! Thank you :)